src/guides/v2.4/release-notes/commerce-2-4-2.md

```diff
@@ -1419,12 +2031,30 @@ We have fixed hundreds of issues in the Magento 2.4.2 core code.
 
 *  Shoppers can now successfully move a product from one wishlist to another.
 
-## Known issues
+<!-- ENGCOM-8062 -->
 
-**Issue**: Discounts for a specific payment method persist during checkout for a purchase order even when the buyer changes payment method during final checkout. As a result, customers may receive a discount that they are not entitled to. This occurs because a cart rule for the original payment method is still applied despite the change in payment method. **Workaround**: None. <!-- KB-827 -->
+*  Magento now displays selected configurable product attribute options in the wishlist page as expected.  [GitHub-24091](https://github.com/magento/magento2/issues/24091),  [GitHub-22503](https://github.com/magento/magento2/issues/22503)
+
+<!-- ENGCOM-7779 -->
+
+*  Implemented `ActionInterface` for \Magento\Wishlist\Controller\Shared\Allcart. [GitHub-29537](https://github.com/magento/magento2/issues/29537)
+
+## Known issues
 
 **Issue**: The `deleteRequisitionListOutput` query returns details about the deleted requisition list instead of the remaining requisition lists. <!--- MC-39894-->
 
+**Issue**: Shoppers cannot add a configurable product to their cart from a non-default store view in a multi-store deployment. Magento displays this error: `Could not add item to cart. Please check required options and try again`. **Workaround**: None. [GitHub-31660](https://github.com/magento/magento2/issues/31660)<!--- PWA-1298-->
+
+**Issue**: Merchants must activate the Fastly Force TLS functionality from the Admin to enable the global HTTP to HTTPS redirect for all store pages. See the [Redirect HTTP to HTTPS for all pages on Cloud (Force TLS)](https://support.magento.com/hc/en-us/articles/360006296953) Knowledge Base article. <!--- MC-39988-->
+
+### B2B known issues
+
+**Issue**: B2B buyers can use online payment methods to bypass the usual purchase order flow. This scenario can occur if the buyer can reduce their entire checkout total to a 0 — for example, by a promo code or gift card —  and subsequently remove the code or gift card. Even under those conditions, Magento still places the order for the correct amount based on the prices of the items in their assigned catalog. <!--- B2B-1603-->
+
+**Issue**: Buyers are redirected to the shopping cart when trying to place an order from a purchase order using PayPal Express Checkout when **In-Context Mode** is disabled. <!--- B2B-1604-->
+
+**Issue**: Magento randomly When attempting to create a purchase order and navigating to the initial checkout page, a bad request can occasionally be observed in the browser console when a buyer creates a purchase order and navigates to the checkout page. The buyer can still place the purchase order. Magento displays this error when a buyer has previously created a different purchase order with an online payment method before navigating to checkout page without completing the purchase. **Workaround**: None. <!--- B2B-1605-->
```

Comment:
Something is amiss at the start of this note.

Reply:
Thanks for catching that ugly copy-and-paste. Corrected.